src/components/SendTokensOne.js

```diff
@@ -132,6 +132,25 @@ class SendTokensOne extends React.Component {
     return data;
   }
 
+  /**
+   * Validate inputs and outpus
+   * 1. If inputs were not selected, select inputs from outputs amount
+   * 2. If amount of selected inputs is larger than outputs amount, we create a change output
+   * 3. If inputs were selected, check if they are valid and add address key to input
+   */
+  handleInitialData = (data) => {
```

Comment:
As the docstring is "Validate inputs and outputs", shouldn't we name this method something like "validateInputsAndOutputs"? I feel it is more intuitive.

Reply:
Done.